ush/forecast_postdet.sh

```diff
@@ -84,6 +64,52 @@ FV3_postdet() {
           break
         fi
       done
+    fi  # if [[ "${RERUN}" != "YES" ]]; then
+
+  fi  # if [[ "${warm_start}" == ".true." ]]; then
+
+  #============================================================================
+  # Determine increment files when doing cold start
+  if [[ "${warm_start}" == ".false." ]]; then
+
+    if [[ "${REPLAY_ICS:-}" == "YES" ]]; then
```

Comment:
Without an actual default, bash will whine about `==` being used as a unary operator.
```suggestion
    if [[ "${REPLAY_ICS:-NO}" == "YES" ]]; then
```